push-notifications/basics/ios.md

```diff
@@ -49,18 +83,27 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
         // Anything you want to do in the callback can be added here
     }
 
-    // This will prompt the user for permission to send remote notification
+    let center = UNUserNotificationCenter.current()
+    center.requestAuthorization(options: [.alert, .sound]) { (granted, error) in
+    // This will prompt the user for permission to send remote notification    
     application.registerForRemoteNotifications()
+    
+    // Enable or disable features based on authorization
```

Comment:
Missing closing `}` for `requestAuthorization`, and please update the identical. Other looks fine for me, thanks @david90